start-site/src/test/java/io/spring/start/site/extension/dependency/springshell/SpringShellTestBuildCustomizerTests.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2012-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.spring.start.site.extension.dependency.springshell;
+
+import io.spring.initializr.metadata.BillOfMaterials;
+import io.spring.initializr.metadata.Dependency;
+import io.spring.initializr.web.project.ProjectRequest;
+import io.spring.start.site.extension.AbstractExtensionTests;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class SpringShellTestBuildCustomizerTests extends AbstractExtensionTests {
```

Comment:
Please add some class-level javadoc similar to [this](https://github.com/spring-io/start.spring.io/blob/3c71d57eae02e099b586a0ee38fc572e5a2f571c/start-site/src/test/java/io/spring/start/site/extension/dependency/springsecurity/SpringSecurityTestBuildCustomizerTests.java#L26-L30).

Reply:
@wilkinsona Thanks for holding my hand with this PR.  It has been a pleasure, getting a code review from you!

😄